Userland/Games/FlappyBug/Game.cpp

```diff
@@ -95,6 +95,16 @@ void Game::keydown_event(GUI::KeyEvent& event)
     }
 }
 
+void Game::mousedown_event(GUI::MouseEvent&)
+{
+    if (ready_to_start()) {
```

Comment:
It would be nice to factor the common mouse and keyboard handler code into a `user_flap()` method (or some better name).